README.md

```diff
@@ -6,10 +6,25 @@
 
 Head to the [Discussion Section][mia-community] to join!
 
-If you wish to request the addition of a new component to Mia-Platform marketplace you can use the [Marketplace contribution template][mia-marketplace-contribution].
-
 </div>
 
+# About this space
+
+The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!
```

Comment:
We should add a contributing file to link here, where we explain to not insert clients name or information and other reserved information (e.g. some secret values)

Reply:
That's a good insight, do you have time to write it?

I could bootstrap it but I guess the contributing rules can be a lot @akelity you might want to participate in it as well?

Reply:
I've added the file with a basic structure of paragraph I think may be useful; I did not write them all yet, feel free to add yours